R/glmClassical.R

```diff
@@ -1184,6 +1184,16 @@ GeneralizedLinearModel <- function(jaspResults, dataset = NULL, options, ...) {
           adjust = selectedPvalueAdjustment
         )
       )) }
```

Comment:
very nitpicky, but could you place that "}" on the next line, preceding the "else"? I think that is more in line with our coding style (and makes the if structure more readable).